### PR TITLE
New version: Microsoft.AzureCLI version 2.66.0

### PR DESCRIPTION
**Pull request by PckgrMiranda**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190611)

### CODE REVIEW

**Comment by PckgrMiranda** [comment]
@PckgrBot

File: manifests/m/Microsoft/AzureCLI/2.66.0/Microsoft.AzureCLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  InstallerSha256: 6E75D4381481F516CC0D0DC558EC9A39307B76601FCA3646747AF5ACB9560433
+  ProductCode: '{23E32D4E-81FD-463B-8C4F-F9D58FA7E748}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Microsoft Azure CLI (64-bit)
+    Publisher: Microsoft Corporation
+    ProductCode: '{23E32D4E-81FD-463B-8C4F-F9D58FA7E748}'
+    UpgradeCode: '{90762FEC-9554-4729-A107-C6A8EA316698}'
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles%\Microsoft SDKs\Azure\CLI2'
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/m/Microsoft/AzureCLI/2.66.0/Microsoft.AzureCLI.installer.yaml:33]:
```suggestion
- Architecture: x86
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> ##[error] Manifest Error: Duplicate installer entry found.

Guessing that the entry with `32-bit` in the name might have `x86` architecture.

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/Microsoft/AzureCLI/2.66.0/Microsoft.AzureCLI.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  InstallerUrl: https://github.com/Azure/azure-cli/releases/download/azure-cli-2.66.0/azure-cli-2.66.0.msi
+  InstallerSha256: 6E1DD47EE3EB2480FAD5DB2F0527C56495E7739BE64C825A53110E4684F23578
+  ProductCode: '{AEDA561A-D438-4734-AD5E-E751C641379F}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Microsoft Azure CLI (32-bit)
+    Publisher: Microsoft Corporation
+    ProductCode: '{AEDA561A-D438-4734-AD5E-E751C641379F}'
+    UpgradeCode: '{DFF82AF0-3F95-4AC9-8EFD-948604FDB028}'
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles(x86)%\Microsoft SDKs\Azure\CLI2'
```
Comment (stephengillie) [inline manifests/m/Microsoft/AzureCLI/2.66.0/Microsoft.AzureCLI.installer.yaml:21]:
```suggestion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Apparently the `x86` installer is already described. This manifest appears to have 3 installer entries for 2 installers.

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by o-l-a-v** [comment]
@stephengillie 

The portable ZIP was left out in this PR for v2.66.0. It's included in the Azure CLI release:

* <https://github.com/Azure/azure-cli/releases/tag/azure-cli-2.66.0>

And the WinGet manifest for v2.65.0 had ZIP:

* <https://github.com/microsoft/winget-pkgs/blob/master/manifests/m/Microsoft/AzureCLI/2.65.0/Microsoft.AzureCLI.installer.yaml>

Maybe the automation for reviewing PRs could get a test that checks new manifest version got at least as many installer types and architectures as the previous version? Here it'd be three:

* x64 MSI
* x64 ZIP
* x86 MSI

**Comment by stephengillie** [comment]
Hi @o-l-a-v,

This would be a good [New Issue](https://github.com/microsoft/winget-pkgs/issues).

**Comment by stephengillie** [comment]
@o-l-a-v and @Trenly,

I'm very sorry for:
- Failing to process warning labels meant to prevent merging of incomplete PRs.
- Requesting to submit a feature that is already in place.
- Being glib while doing so. 

I have fixed the processes in my automation, and will adjust my attitude.

**Comment by Trenly** [comment]
> @o-l-a-v and @Trenly,
> 
> I'm very sorry for:
> 
> * Failing to process warning labels meant to prevent merging of incomplete PRs.
> * Requesting to submit a feature that is already in place.
> * Being glib while doing so.
> 
> I have fixed the processes in my automation, and will adjust my attitude.

No need to apologize @stephengillie - You didn't seem glib at all. There was absolutely nothing wrong with asking for a feature request, especially since it's hard to keep track of everything that goes on in such an active repo like this. You're just one person, don't be too hard on yourself!!

**Pull request merged**